### PR TITLE
core&ui: add manage user priv

### PR DESCRIPTION
**Pull request by pandadtdyy**

### CODE REVIEW

File: packages/ui-default/pages/manage_user_priv.page.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  async function handleOpenUserPrivDialog(ev) {
+    const uid = ev.target ? $(ev.target).data('uid') : ev._id;
+    const priv = ev.target ? $(ev.target).data('priv') : ev.priv;
+    await setPrivDialog.clear(priv);
+    const action = await setPrivDialog.open();
+    if (action !== 'ok') return;
+    let userPriv = 0;
+    setPrivDialog.$dom.find('input.priv[type=checkbox]').each((i, e) => {
+      if (e.checked) userPriv |= 1 << e.value;
```
Comment (pandadtdyy) [inline packages/ui-default/pages/manage_user_priv.page.js:58]: Here is a wrong calc

File: packages/ui-default/pages/manage_user_priv.page.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    });
+    changeUserPriv(uid, userPriv);
+  }
+
+  async function handleClickSelectUser() {
+    const action = await selectUserDialog.clear().open();
+    if (action !== 'ok') {
+      return;
+    }
+    const user = await fetch(`/user/${selectUserSelector.value()._id}`, {
```
Comment (undefined-moe) [inline packages/ui-default/pages/manage_user_priv.page.js]: use `request()`

File: packages/ui-default/templates/manage_user_priv.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            {%- endfor -%}
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="section">
+  <div class="section__header">
+    <h1 class="section__title">{{ _('User Priv') }}</h1>
```
Comment (undefined-moe) [inline packages/ui-default/templates/manage_user_priv.html]: `User Privilege`

File: packages/ui-default/templates/manage_user_priv.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        <col class="col--uid">
+        <col class="col--user">
+        <col class="col--priv">
+        <col class="col--actions">
+      </colgroup>
+      <thead>
+        <tr>
+          <th class="col--uid">{{ _('User ID') }}</th>
+          <th class="col--user">{{ _('Username') }}</th>
+          <th class="col--priv">{{ _('Priv') }}</th>
```
Comment (undefined-moe) [inline packages/ui-default/templates/manage_user_priv.html]: Privilege

File: packages/ui-default/templates/manage_user_priv.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+          <th class="col--actions">{{ _('Actions') }}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {%- for udoc in udocs -%}
```
Comment (undefined-moe) [inline packages/ui-default/templates/manage_user_priv.html]: add another line `default` which controls `default.priv` in system setting

Comment (undefined-moe) [reply packages/ui-default/templates/manage_user_priv.html]: Also this setting should change all user priv that equals to previous default priv

**Review by undefined-moe — APPROVED** [review]

**Pull request merged**